FiraMono-4.002/LICENSE

```diff
@@ -1,8 +1,12 @@
-Copyright (c) 2012-2014, The Mozilla Foundation and Telefonica S.A.
+Copyright (c) 2014, Mozilla Foundation https://mozilla.org/
+with Reserved Font Name Fira Sans.
 
-This Font Software is licensed under the SIL Open Font License, Version 1.1
-or any later version.
+Copyright (c) 2014, Mozilla Foundation https://mozilla.org/
+with Reserved Font Name Fira Mono.
```

Comment:
The weight part of the font name doesn't need to be reserved.  (the FAQ has a little blurb on that: http://scripts.sil.org/OFL-FAQ_web#7fba289b)